### PR TITLE
Adding box.json for so installation can work directly from CommandBox.

### PR DESCRIPTION
**Pull request by andrew-dixon**
@mattlevine The addition of this box.json file will allow installation via CommandBox and will allow Mura to be added to the Forgebox repository.

### CODE REVIEW

File: box.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+    "name":"Mura CMS",
+    "version":"",
+    "author":"Blue River Interactive Group, Inc.",
+    "location":"https://github.com/blueriver/MuraCMS/archive/master.zip",
+    "directory":"",
+    "createPackageDirectory":true,
```
Comment (bdw429s) [inline box.json]: I would set this to false if this package is meant to be a full application that gets dumped directly in the folder.  Having this set to the default of true will put everything in the repo in a sub folder named after the slug (which is blank right now).

Comment (mattlevine) [inline box.json]: Thanks for the heads up!

**Comment by andrew-dixon** [comment]
@bdw429s Thanks for taking a look.

**Comment by andrew-dixon** [comment]
@mattlevine I've just tested this using:

`install https://github.com/blueriver/MuraCMS/archive/develop.zip`

and then

`start`

from inside CommandBox from an empty directory and it appears to have worked ok. It opened in the browse with the Mura install screen.

**Comment by bdw429s** [comment]
Cool, should work with this as well:
```
Install blueriver/MuraCMS
```

**Pull request merged**